windows/translate/installerlng.nsi

```diff
@@ -10,3 +10,6 @@
 
 ; !insertmacro MUI_LANGUAGE "Italian"
 ; !include "translate\it.nsi" ; include italian
+
+!insertmacro MUI_LANGUAGE "Dutch"
```

Comment:
I'm not sure if "Dutch" is correct as language/value here? I will try to compile everything and see if it works ;-)

Reply:
Yes. It works. Great!